src/components/ModalPayment/ModalConfirmation/ModalConfirmation.tsx

```diff
@@ -46,7 +46,7 @@ const ModalConfirmation = (props: Props) => {
   const confirmationText = (purchaseType, sellerName, amount) => {
     switch (purchaseType) {
       case ModalPaymentTypes.modalPages.donation:
-        return `We appreciate your support. We'll let you know when ${sellerName} receives your donation!`;
+        return `We appreciate your support. We'll email you your voucher when ${sellerName} opens back up!`;
```

Comment:
I think this is the updated confirmation text is
https://www.figma.com/file/I0XPeX6re8t14J4nZOvVNp/Send-Chinatown-Love?node-id=104%3A1125  from https://trello.com/c/EgneaM8N/750-voucher-check-out-page-and-receipt-needs-to-be-updated

Also a future todo: translations for the confirmation text

Reply:
I see, I just checked, the voucher Thank you page already uodated. So we don't need to update here. Correct me if I'm wrong. : )

Reply:
Unless I misinterpreted, I thought the confirmation text for `donation` is `We appreciate your support for ${sellerName}! You will receive an an email receipt shortly with the details of your donation`

It doesn't seem to make sense to have a confirmation telling users when the seller opens back for a one-off donation

Reply:
AH Apologies! let's just switch out the word "donation" for "voucher" 

"We appreciate your support of Shunfa Bakery! You will receive an email receipt shortly with the details of your voucher."

Reply:
I think to streamline things, we can keep the same language but have the word "donation". 

We decided to remove the message about a seller opening back up because all our merchants have opened already (unless they permanently closed)

Reply:
I think this should read. For donations:

`We appreciate your support for ${sellerName}! You will receive an an email receipt shortly with the details of your donation.`

for vouchers:

`We appreciate your support for ${sellerName}! You will receive an an email receipt shortly with the details of your voucher`